src/pages/EditClub.js

```diff
@@ -36,74 +38,95 @@ const EditClub = ({bookClubs, updateBookClub, memberships, currentUser}) => {
   }
   return (
     <>
-      <h2>Edit Club</h2>
-      <Form>
-        <FormGroup>
-          <Label for="name">
-            Name
-          </Label>
-          <Input
-            id="name"
-            name="name"
-            type="text"
-            onChange={handleChange}
-            value={editBookClub.name}
+      <h2 className='title-new-account'>Edit Club</h2>
+      <div className='edit-container'>
+        <div className='form-images'>
+          <img
+            src={superhero}
+            width={'325px'}
+            height={'400px'}
+          /> 
+          <Form>
+            <FormGroup>
+              <Label for="name" className=''>
+                Name
+              </Label>
+              <Input
+                id="name"
+                name="name"
+                type="text"
+                onChange={handleChange}
+                value={editBookClub.name}
+                />
+            </FormGroup>
+            <FormGroup>
+              <Label for="summary">
+                Summary
+              </Label>
+              <Input
+                id="summary"
+                name="summary"
+                type="text"
+                onChange={handleChange}
+                value={editBookClub.summary}
+              />
+            </FormGroup>
+            <FormGroup>
+              <Label for="meeting_dates">
+                Meeting Dates
+              </Label>
+              <Input
+                id="meeting_dates"
+                name="meeting_dates"
+                type="text"
+                onChange={handleChange}
+                value={editBookClub.meeting_dates}
+                />
+            </FormGroup>
+            <FormGroup>
+              <Label for="book_of_the_month">
+                Book of the Month
+              </Label>
+              <Input
+                id="book_of_the_month"
+                name="book_of_the_month"
+                type="text"
+                onChange={handleChange}
+                value={editBookClub.book_of_the_month}
+                />
+            </FormGroup>
+            <FormGroup>
+              <Label for="book_of_the_month_picture">
+                  URL for book of the month picture
+              </Label>
+              <Input
+                id="book_of_the_month_picture"
+                name="book_of_the_month_picture"
+                type="text"
+                onChange={handleChange}
+                value={editBookClub.book_of_the_month_picture}
+                />
+            </FormGroup>
+            <NavLink to='/yourclubs'>
+            <Button 
+            name="submit"
+            onClick={handleSubmit} 
+            style={  
+              {backgroundColor: '#ff0000',
+              color: 'black'
+            }}
```

Comment:
indent

Reply:
also everything will need to be properly nested with the button which is inside the NavLink